packages/edit-site/src/hooks/commands/use-edit-mode-commands.js

```diff
@@ -83,6 +83,37 @@ function usePageContentFocusCommands() {
 	return { isLoading: false, commands };
 }
 
+function useEditorModeCommands() {
+	const { switchEditorMode } = useDispatch( editSiteStore );
+	const { canvasMode, editorMode } = useSelect(
+		( select ) => ( {
+			canvasMode: unlock( select( editSiteStore ) ).getCanvasMode(),
+			editorMode: select( editSiteStore ).getEditorMode(),
+		} ),
+		[]
+	);
+
+	if ( canvasMode !== 'edit' ) {
+		return { isLoading: false, commands: [] };
+	}
+
+	const commands = [];
+
+	if ( editorMode === 'text' ) {
```

Comment:
Can we move this check above (line 96 as well

Reply:
Where do you think? I'm following the same structure as `usePageContentFocusCommands` right above.

Reply:
I mean something like https://github.com/WordPress/gutenberg/pull/53496/files#r1290649645 to avoid the two conditions, since this loader only return a single command anyway.